start.sh

```diff
@@ -0,0 +1,29 @@
+arch=$(uname -m)
```

Comment:
Would it make sense to provide a startup script (.bat) for Windows users?

Reply:
The updated README has additional instructions for Windows to install WSL (Windows Subsystem for Linux which basically an additional layer that allow users to run Linux in Windows, providing tools and application etc. WSL can run shell script smoothly without any issues.